thriftbp/ttl_client_test.go

```diff
@@ -26,3 +26,23 @@ func TestTTLClient(t *testing.T) {
 		t.Error("Expected IsOpen call after sleep to return false, got true.")
 	}
 }
+
+func TestTTLClientNegativeTTL(t *testing.T) {
```

Comment:
Is this flaky?  In my experience, timing-related tests usually are.  Does it pass with `--count=1000`?

Reply:
it's not flaky as `time.Sleep` guarantees to sleep at least the desired amount of time (it would be flaky if `time.Sleep` sleeps slightly less than the desired time, and also in that case only the other test would be flaky, not this one).